src/utils/get-lybsin-podcasts.js

```diff
@@ -0,0 +1,47 @@
+// Fetches podcast info from https://mongodb.libsyn.com/rss
+
+const RSS_URL = `https://mongodb.libsyn.com/rss`;
+let prodcastList = [];
+
+export const parsePodcasts = podcastXML => {
+    for (var podcast of podcastXML) {
```

Comment:
This is another great use case for the JS `map` function, since we want to take an array of XML podcasts and "map" it to JSON.  [Check this out](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)!